nixos/modules/system/boot/stage-1-init.sh

```diff
@@ -119,6 +119,14 @@ specialMount() {
 }
 source @earlyMountScript@
 
+# Copy initrd secrets staged into /initrd-secrets to their real destinations
+if [ -d "/initrd-secrets" ]; then
+    for secret in $(cd "/initrd-secrets"; find . -type f); do
```

Comment:
```suggestion
    for secret in $(find /initrd-secrets -type f); do
```

Reply:
While awkward, this form is needed because [boot.initrd.secrets](https://nixos.org/manual/nixos/stable/options.html#opt-boot.initrd.secrets) takes full paths for the attribute name and this change roots them under `/initrd-secrets` (e.g. `/initrd-secrets/etc/dropbear/dropbear_rsa_host_key`).

Reply:
This has been addressed with more clarifying comments, resolving conversation...